source/UberRidesTests/RidesClientTests.swift

```diff
@@ -1083,4 +1087,11 @@ class RidesClientTests: XCTestCase {
         let accessToken = ridesClient.fetchAccessToken()
         XCTAssertNil(accessToken)
     }
+    
+    //MARK: Helper
+    func saveFirstRunKey() {
```

Comment:
I think it would make sense to move this whole function to just be part of the setUp() of the test case. With corresponding logic in tearDown() to remove the key